cranelift/filetests/src/zkasm_codegen.rs

```diff
@@ -288,68 +290,154 @@ pub fn compile_clif_function(func: &Function) -> Vec<String> {
     // TODO: I believe it can be done more beautiful way
     let mut funcname = func.name.to_string();
     funcname.remove(0);
-    funcname.push(':');
-    let mut res = vec![funcname];
+    let mut res = vec![format!("{}:", funcname)];
     res.append(&mut lines);
+    res.into_iter()
+        .map(|s| s.replace("label", &format!("label_{}", funcname)))
+        .collect()
+}
+
+/// Builds main for test program
+pub fn build_main(invoke_names: Vec<String>) -> Vec<String> {
```

Comment:
```suggestion
pub fn build_test_main(invoke_names: Vec<String>) -> Vec<String> {
```

seems like a good enough compromise for now that lets us move forward without spinning our wheels on this.